lib/service/invoices.go

```diff
@@ -359,22 +385,35 @@ func (svc *LndhubService) RevertFeeReserve(ctx context.Context, entry *models.Tr
 	return nil
 }
 
-func (svc *LndhubService) AddFeeEntry(ctx context.Context, entry *models.TransactionEntry, invoice *models.Invoice, tx bun.Tx) (err error) {
-	if entry.FeeReserve != nil {
```

Comment:
@kiwiidb why did we check for `entry.FeeReserve` here? can't we check for `invoice.Fee` (new name invoice.RoutingFee) here? 
If the invoice has a not zero routing fee we have to add a fee transaction entry.

Reply:
I guess it can also work but the way how it used to be it's more explicit in avoiding the nil pointer exception.